homeassistant/components/modbus.py

```diff
@@ -33,10 +34,12 @@
 _LOGGER = logging.getLogger(__name__)
 
 SERVICE_WRITE_REGISTER = "write_register"
+SERVICE_WRITE_REGISTERS = "write_registers"
 
 ATTR_ADDRESS = "address"
 ATTR_UNIT = "unit"
 ATTR_VALUE = "value"
+ATTR_VALUES = "values"
```

Comment:
Do you mean that the attribute name "value" should be the same for write_register and write_registers or that i should only have one multimethod called write_register?

Are there any other services that accepts both a single value and an array? Would be good to keep the same standard.